### DISCUSSION TITLE
3D-files

### DISCUSSION THREAD

**Discussion by Soffan76** [pimcore/pimcore#17675] [General]
Does Pimcore support the 3D-format .step?

**Comment by khenow1978** [comment]
no.. it doesn't. I'm facing the same issue right now. You can store the files in Pimcore of course but you can't use thumbnail generator etc. on these files.

**Reply by Soffan76** [reply]
Thank you for your answer. That a disappointment. I guess it will have to be a zip-solution instead then. It would be nice to have the step file both visible and with possibility to download on web.

**Comment by Soffan76** [comment]
It must be some kind of double solution for me.
1. Possibility to see the 3D-model in some kind of viewer. Convert the step-file some how. Tips on converters are very welcome.
2. Possibility to download the step-file as a zip. This already works.
